samples/highcharts/demo/stellar-chart/demo.js

```diff
@@ -0,0 +1,232 @@
+            [1, colors[3]]
+        ]
+    },
+    legend: {
+        enabled: false
+    },
+    pane: {
+        innerSize: '42%',
+        size: '95%',
+        background: [{
+            backgroundColor: '#f7f7f7',
+            borderWidth: 0
+        }, {
+            backgroundColor: '#fff',
+            borderWidth: 0,
+            outerRadius: '42%'
+        }]
+    },
+    xAxis: {
+        tickInterval: 1,
+        min: 0,
+        max: 30,
+        gridLineWidth: 0,
+        labels: {
+            enabled: false
+        },
+        lineWidth: 0
+    },
+    yAxis: {
+        tickInterval: 1,
+        labels: {
+            enabled: false
+        },
+        gridLineWidth: 0.5,
+        gridLineColor: '#BBBAC5',
+        gridLineDashStyle: 'longdash'
+    },
+    tooltip: {
+        borderWidth: 1,
+        backgroundColor: '#fff',
+        shadow: false,
+        outside: true,
+        hideDelay: 20,
+        formatter: function (tooltip) {
+        // Hide tooltip for pie series
+            if (this.series.options.type === 'pie') {
+                return false;
+            }
+            // If not null, use the default formatter
+            return tooltip.defaultFormatter.call(this, tooltip);
+        }
+    },
+    plotOptions: {
+        series: {
+            states: {
+                inactive: {
+                    enabled: false
+                }
+            }
+        },
+        pie: {
+            states: {
+                hover: {
+                    halo: 0
+                }
+            }
+        }
+    },
+    series: [{
+        colorKey: 'custom.stellarMagnitude',
+        maxSize: 14,
+        minSize: 3,
+        tooltip: {
+            pointFormat: '<span class="smaller">Name:</span> {point.name}<br><span class="smaller">Mass:</span> {point.custom.planetMass}<br><span class="smaller">Distance:</span> {point.custom.lightYears} Light Years<br><span class="smaller">Stellar Magnitude:</span> {point.custom.stellarMagnitude}'
+        }
+    }, {
+        type: 'pie',
+        dataLabels: {
+            enabled: false
+        },
+        size: '40%',
+        innerSize: '85%',
+        zIndex: -1,
+        point: {
+            events: {
+                mouseOver() {
+                    const {
+                            minDate,
+                            maxDate
+                        } = this.options,
+                        point = this,
+                        series = this.series,
+                        chart = series.chart,
+                        bubbleSeries = chart.series[0];
+
+                    bubbleSeries.points.forEach(point => {
+                        if (
+                            point.options.custom.discoveryDate < minDate ||
+                            point.options.custom.discoveryDate >= maxDate
+                        ) {
+                            point.graphic.attr({
+                                opacity: 0.2
+                            });
```

Comment:
Does this work? I don't see any change in the opacity of the bubbles?

Reply:
Not working, because of a small mistake with using `minDate` and `maxDate` from the pie series after the requested changes from Kacper. Gonna fix it.